### ISSUE TITLE
When specifying the tags TLK, stacker removes the stacker_namespace tag

### ISSUE THREAD

**Issue by GarisonLotus** [remind101/stacker#496]
This is more of a bother than a real issue.  When a stacker config yaml has tags defined, it removes the default stacker_namespace tag.  This is not necessarily the expected behavior. 

https://github.com/remind101/stacker/blob/ecd8933d715da8ede5b0d6e6d1e98bd827ff6b43/stacker/context.py#L97

When tackling this issue, we have to be cautious of end-users who have workarounds in place to add the stacker_namespace tag back in via the yaml config. Stacker should not crash in this scenario.

**Comment by phobologic** [comment]
Yeah, I think the only tag we would probably add by default is `stacker_namespace` - and we can just skip adding that if it is manually defined already.  Thanks for adding this, @GarisonLotus!
